core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -160,7 +160,7 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
+                } ~ (authenticate(basicauth) | authenticate(certificateAuth)) {
```

Comment:
yes, i have answered `isn't it easier to bypass all authentication this way?` on above 
```
Current auth key auth: auth key ------------>unique Identity object
client certificate auth : namespace(pass by Header) + subject(saved in `CN`)---->unique Identity object
```
Pass namespace by header, we can get unique Identity.